exchangelib/version.py

```diff
@@ -256,10 +263,11 @@ def _get_version_from_service(cls, protocol, api_version):
         log.debug('Response data: %s', r.text)
         try:
             header = to_xml(r.text, encoding=r.encoding).find('{%s}Header' % SOAPNS)
-            if not header:
+            if len(header) < 1:
```

Comment:
`len()` doesn't make much sense because `find()` returns either an XML element or `None`.

`if header is None` is a better alternative because XML elements may evaluate to `False`.